tests/codeboarding_cli/test_incremental_analysis.py

```
@@ -11,136 +11,42 @@
+    assert kwargs["output_dir"] == tmp_path / ".codeboarding"
+    # No change-detection inputs cross the boundary — Core detects internally.
+    assert "base_ref" not in kwargs
+    assert "target_ref" not in kwargs
+    assert "changes" not in kwargs
+    assert "source_sha" not in kwargs
+
+
+def test_incremental_no_git_ref_flags_in_parser() -> None:
+    """The incremental subcommand exposes no git-ref flags — detection is internal."""
+    parser = build_parser()
+    args = parser.parse_args(["incremental", "--local", "/tmp/repo"])
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/repo"

(S108)